sqldelight-studio-plugin/testData/formatting/Formatting_after.sq

```diff
@@ -0,0 +1,33 @@
+import java.util.List;
+import java.util.ArrayList;
+
+CREATE TABLE test (
+    _id INTEGER NOT NULL PRIMARY KEY,
```

Comment:
Yea that's just the default intellij format I don't think I have control
over it. It will do 2 spaces if your preference is 2 spaces for an indent

On Sun, Jul 31, 2016, 2:45 AM Jake Wharton <notifications@github.com> wrote:

> In sqldelight-studio-plugin/testData/formatting/Formatting_after.sq
> <https://github.com/square/sqldelight/pull/431#discussion_r72899059>:
>
> > @@ -0,0 +1,33 @@
> > +import java.util.List;
> > +import java.util.ArrayList;
> > +
> > +CREATE TABLE test (
> > +    _id INTEGER NOT NULL PRIMARY KEY,
>
> four spaces?!?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/square/sqldelight/pull/431/files/79fc87a44b49f376bf12b37575870ff7bd4c6c97#r72899059>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABmQwOja4bgcFt3Wi3zBm5PKezCdU-afks5qbESCgaJpZM4JXkxr>
> .
>